packages/@react-stately/dnd/src/useDroppableCollectionState.ts

```diff
@@ -101,7 +110,7 @@ export function useDroppableCollectionState(props: DroppableCollectionStateOptio
     }
 
     return 'cancel';
-  }, [acceptedDragTypes, getDropOperation, onInsert, onRootDrop, onItemDrop, shouldAcceptItemDrop, onReorder, onDrop]);
+  }, [isDisabled, acceptedDragTypes, getDropOperation, onInsert, onRootDrop, onItemDrop, shouldAcceptItemDrop, onReorder, onDrop]);
 
   return {
```

Comment:
We should return `isDisabled` in this return object too for consistency with `useDraggableCollection` state I think.